src/components/ChapterMap.svelte

```diff
@@ -27,12 +28,14 @@
 
   <legend>
     <div>
-      <span style="background: var(--lightBlue)" /> aktiver Standort
+      <span style="background: var(--lightBlue)" />
+      {$microcopy?.map?.text?.active}
     </div>
     <div>
-      <span style="background: var(--darkGreen)" /> in Gründung
+      <span style="background: var(--darkGreen)" />
+      {$microcopy?.map?.text?.funding}
     </div>
-    <small>Zoomen mit <kbd>ctrl/cmd</kbd></small>
+    <!-- <small>{@html $microcopy?.map?.text?.zoom}</small> -->
```

Comment:
Why remove this?

Reply:
We don't need this anymore. When you try to zoom, the map already shows that you should use cmd+ zoom.